docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -52,9 +61,14 @@ class Sidebar extends React.Component<any, any> {
     document.removeEventListener('keydown', this.handleDocumentKeyDown)
   }
 
-  setSearchInput() {
-    // TODO: Replace findDOMNode with Ref component when it will be merged
-    this._searchInput = (findDOMNode(this) as any).querySelector('.ui.input input')
+  setSearchInput = () => {
+    this._searchInput = this.searchInputRef.current.inputRef.current
+  }
```

Comment:
You don't need this, check this one: https://github.com/stardust-ui/react/pull/1682/files#r307795621